test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_deposit.py

```diff
@@ -114,21 +114,6 @@ def test_invalid_withdrawal_credentials_top_up(spec, state):
     yield from run_deposit_processing(spec, state, deposit, validator_index, valid=True, effective=True)
 
 
-@with_all_phases
-@spec_state_test
```

Comment:
Why is this test wiped?

Reply:
Side effect of discovering (with strong typing) that `index` is a non-existent field now, it was removed. See https://github.com/ethereum/eth2.0-specs/pull/1180#issuecomment-504218583